src/starbunk/bots/__tests__/strategy-loader.test.ts

```
@@ -72,12 +79,37 @@
 		expect(loadedBots[0].defaultBotName).toBe(validBot.name);
 	});
 
+// Mock at the top of the file with other mocks
```

ESLint

[error] Expected indentation of 1 tab but found 0.

(@typescript-eslint/indent)